src/homepage/homepage.html

```diff
@@ -45,14 +95,19 @@
     }
 
     #activities div:nth-child(2n + 1) {
-      background-color: var(--tertiary-color);
+      background-color: var(--primary-color);
     }
 
     #merchandise img {
       width: 450px;
       max-width: calc(100% - 20px);
     }
   </style>
+  <section id="logo">
+    <img src="../../images-optimized/logo.svg" alt="#Trending logo">
+    <span class="title--big">12TH LUSTRUM</span>
+    <span class="title--small">W.I.S.V. 'Christiaan Huygens'</span>
```

Comment:
Some screenreaders skip headers that are not followed by content iirc, so that's not a reason in the first place. Beyond that, Search engine optimization suggests 1 h1, max 2 h2's on a page. Calling everything that is styled big a header is not semantically accurrate. You could opt for a11y tags, but that's out of scope for this specific pr